IntegrationTests/GlobalEvalWithSourceUrlTest.js

```diff
@@ -42,7 +42,8 @@ class GlobalEvalWithSourceUrlTest extends React.Component<{...}> {
         'Expected globalEvalWithSourceUrl to throw on a syntax error',
       );
     }
-    if (!(syntaxError instanceof SyntaxError)) {
+    // Hermes throws an Error, not a SyntaxError
+    if (syntaxError.jsEngine !== 'hermes' && !(syntaxError instanceof SyntaxError)) {
```

Comment:
The exact error is:
> Error: Exception in HostFunction: Compiling JS failed: 1:2:'}' expected at end of block

Is this a bug or a known characteristics of Hermes?

Reply:
@grabbou It's kinda related to a known problem that is not going to be resolved anytime soon. It's not so much of Hermes but the way we are building and linking our apps. I think we can leave the code this way and post an issue on the Hermes repo

Reply:
Okay, thanks, going to do it now!

Reply:
https://github.com/facebook/hermes/issues/400